sagemaker-triton/TensorFlow/Deploy-TensorFlow-Model-Using-NVIDIA-Triton.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>Would change to "We strongly recommend to delete the Real-Time endpoint created"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>You can also mention that we recommend deleting the endpoint to stop incurring cost when finished with the example.</p>

Reply:
Done